src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -119,7 +119,10 @@ public TransactionSubstate Run(EvmState state, IWorldState worldState, ITxTracer
                     {
                         if (_txTracer.IsTracingActions)
                         {
-                            _txTracer.ReportAction(currentState.GasAvailable, currentState.Env.Value, currentState.From, currentState.To, currentState.Env.InputData, currentState.ExecutionType, true);
+                            if (!(currentState.IsTopLevel && currentState.Env.Value == UInt256.Zero))
```

Comment:
This breaks ReportActionEnd/MarkAsSuccess in the tracer as it does not have correlated call.